src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -46,7 +46,12 @@ public void storeMessage(Message message) {
     messageEntity.setProperty("timestamp", message.getTimestamp());
     messageEntity.setProperty("recipient", message.getRecipient());
 
+    if(message.getImageUrl() != null) {
+        messageEntity.setProperty("imageUrl", message.getImageUrl());
+    }
+
     datastore.put(messageEntity);
+
```

Comment:
Thanks - now, can you go through the overall diff (you can see it at https://github.com/jonathantanmy/codeu-spring-2019/pull/22/files) and make sure that no empty lines are added or removed unnecessarily? These are not as bad as unnecessarily indenting or commenting unrelated lines, but can still cause problems during merging.

For example, you added this blank line unnecessarily - please remove it.